components/ui-api-layer/internal/gqlschema/schema.graphql

```diff
@@ -649,6 +649,9 @@ type Mutation {
     enableApplication(application: String!, namespace: String!): ApplicationMapping
     disableApplication(application: String!, namespace: String!): ApplicationMapping
 
+    updatePod(name: String!, namespace: String!, update: JSON!): Pod
```

Comment:
I would replace the name of `update` field with `pod` or something like that - to highlight, that you expect full JSON of the pod.